broker-cli/commands/register.js

```diff
@@ -0,0 +1,40 @@
+const BrokerDaemonClient = require('../broker-daemon-client')
+const { validations, handleError } = require('../utils')
+const { RPC_ADDRESS_HELP_STRING } = require('../utils/strings')
+
+/**
+ * sparkswap id
+ *
+ * Get the BrokerDaemon's Identity
+ *
+ * ex: `sparkswap id`
+ *
+ * @param {Object} args
+ * @param {String} args.publicKey
+ * @param {Object} opts
+ * @param {String} [rpcAddress=null] broker rpc address
+ * @param {Logger} logger
+ */
+
+async function register (args, opts, logger) {
+  const { publicKey } = args
```

Comment:
Im thinking that there is probably a way that `publicKey` can become an option instead of being an argument for the register command... because we already know where the path to this key is.

`sparkswap register` would grab the key from the path that was specified in the configuration file?

Reply:
Yeah I agree with this, the `publicKey` should probably be a path, and have a default to somewhere in the `.sparkswap` directory

Reply:
But to be clear, I think that if we're loading from a path, that should be done within the daemon, not from the CLI. (I think, someone check my thinking on that)